frontend/WikiContrib-Frontend/src/App.css

```diff
@@ -116,17 +116,10 @@ h4.accounts {
   transition: 0.3s;
   font-size: 1.5rem;
   color: #000;
-  outline: none;
```

Comment:
Adjusting the input box's outline was part of another issue, and your PR undid the changes of this PR https://github.com/wikimedia/WikiContrib/pull/100/files. Could you put them back? :)